.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
```

Comment:
Why did we need this file?

Reply:
Please refer to https://editorconfig.org/. In my point of view it is a good way to handle different code styles on public projects.

Reply:
We enforce styles with eslint and stylelint, so we should not need this, I don't think.